frontend/src/routes/home/+page.server.js

```diff
@@ -13,16 +13,16 @@ export const load = async ({ cookies }) => {
     // Get the cookie containing the JWT token
     const jwt = cookies.get('jwt');
 
-    // Send request to the apigateway to check if the user is authenticated
-    const isAuthenticated = await axios.get('http://localhost:3011/verify', {
+    const payload = JSON.parse(Buffer.from(jwt.split('.')[1], 'base64').toString());
```

Comment:
Use `jwt.decode` instead like in `frontend/src/routes/moods/my-mood/[recordId]/thought/+page.server.js`

Reply:
Okay! Done